core/tests/ledger_cleanup.rs

```diff
@@ -275,6 +295,27 @@ mod tests {
         *data_shred_storage_previous = data_shred_storage_now;
     }
 
+    /// Helper function of the benchmark `test_ledger_cleanup_compaction` which
+    /// returns true if the benchmark fails the size limitation check.
+    fn is_exceeded_stop_size_iterations(
+        storage_size: u64,
+        stop_size: u64,
+        exceeded_iterations: &mut u64,
+        iteration_limit: u64,
+    ) -> bool {
+        info!("is_exceeded_stop_size_iterations {} {} {} {}", storage_size, stop_size, &exceeded_iterations, iteration_limit);
+        if stop_size > 0 {
+            if storage_size >= stop_size {
+                *exceeded_iterations += 1;
```

Comment:
What do you think about printing a `warn!()` here? I'm thinking about the case where maybe we don't exceed the iteration_limit, but we may still be interested in to know that we're getting close. I think this could be a good alternative if you agree about removing or demoting the `info!()`